README.md

```diff
@@ -23,6 +23,9 @@ Currently there is only one possible failure (`MissingFieldFailure`) which conta
 It is common that the field has different name in the GPB and in the case class. You can use `GpbName` annotation 
 to override the expected name - see the example.
 
+Sometimes the case class has more fields than the GPB and you just want to ignore them during conversion. You can use `Ignored` 
+annotation to achieve this. Please **note that** the ignored field in the case class **must** have default value!
```

Comment:
Emphasis on "note that" was because I wanted to emphasize the change :) It was not supposed to be added... but emphasis on "must" actually makes sense so leave it :)